Add a review checklist and suggest reviews as a way to get started with the project (#1463)
rust-lang/rustc-dev-guide: src/getting-started.md

[r+]: https://github.com/rust-lang/rust/pull/78133#issuecomment-712726339
[bors]: https://bors.rust-lang.org/queue/rust

### Reviewing PRs

> Another way to contribute to the project is by reviewing existing PRs. We have a [review checklist]

Reviewer: ```suggestion
Reviewing existing PRs helps the project merge new features and bug fixes faster, and everyone is welcome to help! We have a [review checklist]
```

> you can use to get started. We suggest reviewing the PRs of existing contributors. Remember that

Reviewer: I don't think that suggesting reviewing the PRs of existing contributors is essential here; funneling beginners to help each other is very useful to get the easy nits out of the way quickly.

> you can use to get started. We suggest reviewing the PRs of existing contributors. Remember that

Reviewer: ```suggestion
you can use to get started. We suggest reviewing the PRs of existing contributors. Don't worry if you're new; 
```
